examples/eng_wasm_contracts/encryption/src/lib.rs

```diff
@@ -0,0 +1,93 @@
+// Rust’s standard library provides a lot of useful functionality, but assumes support for various
+// features of its host system: threads, networking, heap allocation, and others. SGX environments
+// do not have these features, so we tell Rust that we don’t want to use the standard library
+#![no_std]
+#![allow(unused_attributes)]
+
+#[macro_use]
+extern crate serde_derive;
+extern crate serde;
+// The eng_wasm crate allows to use the Enigma runtime, which provides:
+//     - Read from state      read_state!(key)
+//     - Write to state       write_state!(key => value)
+//     - Print                eprint!(...)
+extern crate eng_wasm;
+
+// The eng_wasm_derive crate provides the following
+//     - Functions exposed by the contract that may be called from the Enigma network
+//     - Ability to call functions of ethereum contracts from ESC
+extern crate eng_wasm_derive;
+
+// The asymmetric features of enigma_crypto
+extern crate enigma_crypto;
+
+// Serialization stuff
+extern crate rustc_hex;
+
+// eng_wasm
+use eng_wasm::*;
+use eng_wasm_derive::pub_interface;
+use eng_wasm::{String, Vec};
+use rustc_hex::ToHex;
+use enigma_crypto::asymmetric::KeyPair;
+
+// State key name "mixer_eth_addr" holding eth address of Mixer contract
```

Comment:
The lines 27, 34 and 35 are commented out, if there is no potential use, maybe it is better to remove them.